fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -4,6 +4,13 @@
 from ....oot_utility import getCustomProperty, ExportInfo
 
 
+def getSceneSettingsOption(scene):
```

Comment:
I don't understand this function

mostly "option" is too vague,
and "scene" is confusing between an oot scene and a blender scene

Reply:
it's to get the proper ``.option`` from either ``ootExportSettings`` or ``ootRemoveSettings`` (idr the exact names), when you remove a scene it's using the export settings instead of the remove settings because it was hardcoded since the other one didn't exist

Reply:
yeah but what is `.option`

Reply:
it's the scene choice from the panel

Reply:
so `getSceneSettingsOption` is like `getSceneSettingsSceneName` ?

Reply:
I just called it ``getSceneSettingsOption`` because of the name of the things to get the scene value but yeah I guess scene name makes more sense

Reply:
gonna name it ``getSceneNameSettings`` for "get scene name from settings" it should be better this way